docker/mu-plugins/jetpack-debug-helper/inc/class-broken-token-xmlrpc.php

```diff
@@ -67,17 +92,39 @@ public function render_ui() {
 			<div id="current_xmlrpc_errors">
 
 
-				<form action="<?php echo esc_url( admin_url( 'admin-post.php' ) ); ?>" method="post">
```

Comment:
Minor given this is the helper plugin, but why the change? `esc_url` should be used anyplace an URL is used, inclusive of attribute nodes:
https://developer.wordpress.org/reference/functions/esc_url/

Reply:
I sincerely don't remember making this change. You're right. reverting this (in two places) here https://github.com/Automattic/jetpack/pull/16194/commits/d316b9211e7067680552e385858145ad90ff8ef1